airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -104,7 +104,7 @@ def __init__(
         self.use_legacy_sql = use_legacy_sql
         self.location = location
         self.running_job_id = None  # type: Optional[str]
-        self.api_resource_configs = api_resource_configs if api_resource_configs else {}  # type Dict
+        self.api_resource_configs = api_resource_configs if api_resource_configs else {}  # type dict
```

Comment:
Can we:
```suggestion
        self.api_resource_configs: Dict = api_resource_configs if api_resource_configs else {}
```
Or there is reason we keep type in the comment? Same for other places.